### ISSUE TITLE
Install Error: missing autoconf (following documentation)

### ISSUE THREAD

**Issue by raulium** [ntop/nDPI#676]
Running macOS Mojave 10.14.3
Xcode-select version 2354

Following README.md steps to reproduce:
```
$ git clone https://github.com/ntop/nDPI.git
$ cd nDPI
$ ./autogen.sh
autoconf is missing: please install it and try again
```

Documentation in /doc/nDPI_QuickStartGuide.pdf provided no guidance. And [website instructions](https://www.ntop.org/get-started/download/) are out of date/incorrect.

should include `brew install autoconf automake` as it does not come included by default on Mac systems.

**Comment by rizwanrazi** [comment]
is it fixed?

**Comment by raulium** [comment]
Not particularly, but the issue was mostly with the documentation -- though I can't remember why I closed it the same day. If you want to install nDPI you just need to have homebrew already installed and ensure you've done something like:

`brew install autoconf automake pkg-config json-c`

I just recreated the steps myself and found I also needed the other two -- but that might not be the definitive list of packages not already installed on macOS.

Disclaimer: I've since upgraded to macOS 12.2.1, running on an Apple M1, and xcode-select version 2392.
